Lib/sqlite3/__init__.py

```diff
@@ -55,3 +55,15 @@
 """
 
 from sqlite3.dbapi2 import *
+from sqlite3.dbapi2 import (_deprecated_names,
+                            _deprecated_version_info,
+                            _deprecated_version)
+
+
+def __getattr__(name):
+    if name in _deprecated_names:
+        from warnings import warn
+        warn(f"{name} is deprecated and will be removed in Python 3.14",
```

Comment:
Style (linebreak after local import)
```suggestion

        warn(f"{name} is deprecated and will be removed in Python 3.14",
```